nucleus/deploy/nucleus_integration.py

```diff
@@ -26,7 +26,7 @@ class NucleusDatasetInferenceRun:
     # For the demo, we will need our Nucleus Dataset to have `image_location`s in s3://scale-ml-hosted-model-inference
     def __init__(
         self,
-        hmi_async_job: AsyncModelEndpointResponse,
+        hmi_async_job: AsyncModelEndpointBatchResponse,
```

Comment:
Not really part of this PR but maybe we can take a pass at removing all "hmi" stuff from user-facing names?